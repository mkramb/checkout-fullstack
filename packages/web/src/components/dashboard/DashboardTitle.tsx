import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';

export interface DashboardTitleProps {
  readonly title: string;
}

export const DashboardTitle = ({ title }: DashboardTitleProps) => (
  <AppBar position="static">
    <Toolbar>{title}</Toolbar>
  </AppBar>
);
