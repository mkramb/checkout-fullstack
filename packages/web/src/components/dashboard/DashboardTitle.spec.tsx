import React from 'react';
import { DashboardTitle, DashboardTitleProps } from './DashboardTitle';
import { shallow } from 'enzyme';

describe('DashboardTitle', () => {
  const renderComponent = (props: DashboardTitleProps) => {
    return shallow(<DashboardTitle {...props} />);
  };

  it('should pass', () => {
    expect(renderComponent({ title: 'Feedback Dashboard' })).toMatchSnapshot();
  });
});
