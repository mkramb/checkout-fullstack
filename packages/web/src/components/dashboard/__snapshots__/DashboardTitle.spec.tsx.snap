// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardTitle should pass 1`] = `
<WithStyles(ForwardRef(AppBar))
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    Feedback Dashboard
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
